.github/workflows/ci.yml

```diff
@@ -112,6 +112,16 @@ jobs:
       - name: carthage build
         run: ./script/carthage.sh build --cache-builds --no-skip-current
 
+  Danger:
```

Comment:
I think this could live in a separate action instead, like `danger.yml` which only runs on PRs